_posts/2015-10-13-scrum-basics-1.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Scrum Basics Series - Part 1: What is Scrum?"
+date: "2015-10-13 08:23:00"
+icon: university
+categories: agile
+authors: ["Anja B."]
+---
+
+As a Scrum Master I’m often asked what Scrum is and what I’m doing the whole day long.
+Well, part of my job is to teach new colleagues exactly that.
+Now we'd like to share this knowledge with you in our **Scrum Basics Series**.
+This series will consist of 6 parts and starts with the basic question:
+What is Scrum?
+
+So let’s start with a short definition:
+
+>“Scrum is a team based framework
+to develop complex products
+or systems in an iterative way.“
+
+# What does that mean in detail?
+
+In Scrum you are always working in a team.
+This team should be cross-functional.
+It should have all the abilities needed to fulfil the tasks, e.g. in software development a team consists of backend developers and frontend developers as well as designers and testers.
+In an ideal world it would go from creating concepts, over developing and testing the product, to rollout, marketing and sales.
+But most companies “only” have the development part in their Scrum teams.
+The members of those teams should strive to get “T-shaped”.
+That means they not only have one field of expertise, but also a broad basis so they can help out in other areas in the team.
+That doesn’t mean everyone should know everything, it only means everyone is willing to do more than just their own thing.
+
+# Advantages
+
+The big advantage of those teams in comparison to assigning single persons to specific parts of the product development is, that you always have all the abilities needed in one room.
+In [waterfall](https://en.wikipedia.org/wiki/Waterfall_model) you often have the problem that the person who did the backend development is already in the next project when the designers realise that they need some more data from the backend.
+In Scrum that won’t happen.
+Another advantage is that you always have several eyes on one problem.
+Solutions can be found faster if you have more experience in the room and people who get stuck at some point always have someone to ask right next to them.
+And don’t forget the much faster feedback cycle with the testers when they are right inside the team!
```

Comment:
And let's not forget the much faster feedback cycle with the testers when they are right inside the team!